aws/data_source_aws_ec2_instance_type_test.go

```diff
@@ -0,0 +1,101 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceAwsEc2InstanceType_attributes(t *testing.T) {
```

Comment:
@lde This is very close! Excellent work. This is the last thing I see that needs work.

Let's split this one test, `TestAccDataSourceAwsEc2InstanceType_attributes`, into 5 tests:

1. `TestAccDataSourceAwsEc2InstanceType_basic`
2. `TestAccDataSourceAwsEc2InstanceType_metal`
3. `TestAccDataSourceAwsEc2InstanceType_gpu`
4. `TestAccDataSourceAwsEc2InstanceType_fpga`
5. `TestAccDataSourceAwsEc2InstanceType_accelerator`

The basic test should test as many attributes as possible. The other should have only attributes that are specific to those types.

Reply:
Hi @YakDriver, I've splited tests and added somes missing attributes.